enriched-span-constants/build.gradle.kts

```diff
@@ -60,7 +60,7 @@ sourceSets {
 dependencies {
   api("com.google.protobuf:protobuf-java-util:3.12.2")
 
-  implementation("org.hypertrace.core.datamodel:data-model:0.1.8")
+  implementation("org.hypertrace.core.datamodel:data-model:0.1.9-SNAPSHOT")
```

Comment:
SNAPSHOT?

Reply:
This change is dependent on another change. Once that is merged, this will be updated.